javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,68 @@
+// Flight booking fullname function
+
+const useFormalName = "Lord" + getFullname;
+
+getFullname("Benjamin", "Hughes", useFormalName);
+let fullname1  = "Benjamin";
+let fullname2  = "Hughes";
+function getFullname(fullname1, fullname2) {
+  if (useFormalName) {
+  return ("Lord" + " " + fullname1 + " " + fullname2)
+} else {
+  return (fullname1 + " " + fullname2)
+}
+}
+getFullname(fullname1, fullname2);
+/* the getFullname for women will work the similar way: we need to create boolean variable useWomenName and if it's true
+we should use Queen as a formal way instead of Lord.
+*/
+
+// Event application
+function getEventWeekday(){
+let days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
+const todaysDate = new Date();
+
+}
+// Weather wear
+
+function whatToWear(a) {
+  if (a >= 0 && a <= 15) {
+    console.log("Wear jeans and a jacket")
+  } else if (a >= 16) {
+    console.log("Wear top and shirts")
+  } else {
+    console.log("Wear a space suit - it is too cold!")
+  }
+}
+const clothesToWear = whatToWear(5);
+console.log(clothesToWear);
+
+// Student manager
+
+const class07Students = [];
+const queenName = "Queen";
+function addStudentToClass(studentName) {
+for (let i = 0; i < 6; i++) {
```

Comment:
You actually fill up the class with just one student here. You add a student 6 times, filling up the class. You instead should just add the student to the array, but only if the student is not already in the class, and also only if the class has less than 6 students already. Unless if `studentName === queenName`, then she gets added even if there are already 6 students. But she only gets added once, so if she is already in the class, she will not be added. 👉